HOWTO_RELEASE.md

```diff
@@ -0,0 +1,169 @@
+# Release Howto
+
+This document is intended to give precise instructions on the process of making a release. Its purpose is to avoid broken packages, broken documentation and many other things that can go wrong in the release process. Since this is not everyday work and may be done under the stress of a (self-imposed) deadline, it is clearly beneficial to have a checklist to hold on to.
+
+**Note:** The instructions in this document are tentative until tested in practice. They are also written from the perspective of Linux and may need adaption for other platforms.
+
+## 1. Agree on a release schedule
+
+This involves the "what" and "when" of the release process and fixes a feature set that is supposed to be included in the new version. The steps are:
+
+- Open an issue on the issue tracker using the title **Release x.y.z**.
+- Discuss and agree on a set of open PRs that should be merged before making a release.
+
+## 2. Make sure tests succeed and docs can be built
+
+When all required PRs are merged, ensure that the latest `master` branch is sane. Travis CI checks every PR, but certain things like CUDA cannot be tested there and must therefore undergo tests on a local machine, for at least Python 2.7 and one version of Python 3.
+
+- Make a new test conda environment and install all dependencies:
+  ```
+  $ conda create -n release35 python=3.5 nomkl numpy scipy future matplotlib pytest
```

Comment:
You mean remove the dollar signs? I think we did that in the installing doc, too.